azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -1689,12 +1691,14 @@ public int getExecutableFlows(int projectId, String flowId, int from,
         24 * 60 * 60 * 1000;
 
     private final long executionLogsRetentionMs;
+    private final long cleanerThreadCustomWaitIntervalMs;
```

Comment:
you can name it cleanerThreadWaitIntervalMs only, it is either default or custom

Reply:
changed